### PR TITLE
Alttext5

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

**Comment by v-emilypr** [comment]
@Rick-Anderson, @mcleblanc
This PR addresses missing alt text and is ready for final review.

**Comment by Rick-Anderson** [comment]
@v-emilypr  can you check the build report?
https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C16%5C90f071cf-c002-b629-5e11-5bc968bb67ca%5CPullRequest%5C202203161841516918-672%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-16T18%3A46%3A44Z&sr=b&sp=r&sig=3VkF46sStnWEUjFA5h52sTX6WmMIL8H%2Ba4NlqOozDuw%3D

image

File: iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/using-failed-request-tracing-to-troubleshoot-classic-asp-errors.md [hunk 1]
```diff
@@ -103,13 +103,13 @@ When you use Internet Explorer to open a failed request tracing log file, a vari
 
 In the **Errors and Warning** section of summary, you will see a **view trace** link, as shown in the following illustration:
 
-[![](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image2.jpg)](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image1.jpg)
+![Screenshot of a browser window displaying the Request Summary of a failed trace log.](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image1.jpg)
```
Comment (Rick-Anderson) [inline iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/using-failed-request-tracing-to-troubleshoot-classic-asp-errors.md:106]: Line 106: [Warning-[PR only](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/pr-only?branch=main)] Alt text 'Screenshot of a browser window displaying the Request Summary of a failed trace log.' is duplicated. Within an article, alt text must be unique.

File: iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/using-failed-request-tracing-to-troubleshoot-classic-asp-errors.md [hunk 2]
```diff
@@ -155,13 +155,13 @@ As shown in the previous example, when you use Internet Explorer to open a faile
 
 In the **Errors and Warning** section of summary, you will see a **view trace** link, as shown in the following illustration:
 
-[![](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image6.jpg)](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image5.jpg)
+![Screenshot of a browser window displaying the Request Summary of a failed trace log.](using-failed-request-tracing-to-troubleshoot-classic-asp-errors/_static/image5.jpg)
```
Comment (Rick-Anderson) [inline iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/using-failed-request-tracing-to-troubleshoot-classic-asp-errors.md:158]: Line 158: [Warning-[PR only](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/pr-only?branch=main)] Alt text 'Screenshot of a browser window displaying the Request Summary of a failed trace log.' is duplicated. Within an article, alt text must be unique.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Please fix build warnings.

**Comment by v-emilypr** [comment]
@Rick-Anderson, @mcleblanc
The duplicated text has been corrected and is ready for final review. My apologies on the delay in resolving as I was out of the office. Thanks :)

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**